contribs/gmf/src/directives/print.js

```diff
@@ -344,6 +363,12 @@ gmf.PrintController = function($rootScope, $scope, $timeout, $q, $injector,
   this.rotation = 0;
 
   /**
+   * @type {Array.<string>}
+   * @export
+   */
+  this.hiddenAttributes;
```

Comment:
Might be better to name it `hiddenAttributeNames` to distinguish with `attributesOut` which are not an array of strings.